### ISSUE TITLE
[Bug] Error when invoking MagmaDelete

### ISSUE THREAD

**Issue by matthewsia98** [dccsillag/magma-nvim#52]
**Describe the bug**
Error when invoking MagmaDelete
```
Error detected while processing function remote#define#CommandBootstrap[5]..remote#define#request:              
line    2:                                                                                                      
Error invoking '/home/siam/.dotfiles/nvim/plugged/magma-nvim/rplugin/python3/magma:command:MagmaDelete' on chann
el 3 (python3-rplugin-host):                                                                                    
error caught in request handler '/home/siam/.dotfiles/nvim/plugged/magma-nvim/rplugin/python3/magma:command:Magm
aDelete []':                                                                                                    
Traceback (most recent call last):                                                                              
  File "/home/siam/.dotfiles/nvim/plugged/magma-nvim/rplugin/python3/magma/utils.py", line 14, in inner         
    func(self, *args, **kwargs)                                                                                 
  File "/home/siam/.dotfiles/nvim/plugged/magma-nvim/rplugin/python3/magma/__init__.py", line 265, in command_de
lete                                                                                                            
    magma.delete_cell()                                                                                         
  File "/home/siam/.dotfiles/nvim/plugged/magma-nvim/rplugin/python3/magma/magmabuffer.py", line 171, in delete_
cell                                                                                                            
    self.update_interface()                                                                                     
  File "/home/siam/.dotfiles/nvim/plugged/magma-nvim/rplugin/python3/magma/magmabuffer.py", line 179, in update_
interface                                                                                                       
    self.clear_interface()                                                                                      
  File "/home/siam/.dotfiles/nvim/plugged/magma-nvim/rplugin/python3/magma/magmabuffer.py", line 151, in clear_i
nterface                                                                                                        
    self.outputs[self.selected_cell].clear_interface()                                                          
KeyError: <magma.utils.Span object at 0x7ff90094fa30> 
```

**To Reproduce**
1. Select range and invoke MagmaEvaluateVisual OR invoke MagmaEvaluateLine
2. Invoke MagmaDelete while cell is selected
3. See error

**Expected behavior**
Delete the cell

**Screenshots**
<img width="326" alt="image" src="https://user-images.githubusercontent.com/71731492/185716152-73cb4426-92be-4ee0-a234-dbfed4a7d2a0.png">
<img width="960" alt="image" src="https://user-images.githubusercontent.com/71731492/185716181-b259ce3b-4cc3-404f-8be4-c14f5268f01b.png">
<img width="463" alt="image" src="https://user-images.githubusercontent.com/71731492/185716417-5bdb854d-d3a2-4397-addb-338b5f4e2c56.png">

**Desktop**
 NVIM v0.8.0-dev-888-g7ff283b1b
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3

**Comment by fe-lipe-git** [comment]
Having the same issue since yesterday, when I updated the plugin.

**Comment by MrBober** [comment]
I have the same issue
You can go back to commits before `edfd16e065f8965a492e0a3d078a713aab096ffc` (which is the commit which broke it)
The newest commit before that one was: `edf60f0b809c88dab3a4ea492e0dcf80fc00af5b`
so you can just go to `autoload/plugged/magma-nvim` and run `git checkout edf60f0b809c88dab3a4ea492e0dcf80fc00af5b`

**Comment by MrBober** [comment]
So... I found a problem, basically sometimes it was trying access a key in `self.outputs` while it was empty, so I added checks for that and it fixed the issue

**Comment by dccsillag** [comment]
Solved in 395b48e2e202d82fca76c15d2dcd8785c125d686 (via #58). Thanks, @MrBober!
